README.md

```diff
@@ -197,38 +197,40 @@ This also works if the `d2l-labs-filter-dropdown` is a child in the shadow-dom o
 <d2l-labs-filter-dropdown id="filter"> ... </d2l-labs-filter-dropdown>
 ```
 
-## Developing, Testing and Contributing
+## Developing
 
 After cloning the repo, run `npm install` to install dependencies.
 
-If you don't have it already, install the [Polymer CLI](https://www.polymer-project.org/3.0/docs/tools/polymer-cli) globally:
+### Running the demos
 
-```shell
-npm install -g polymer-cli
-```
-
-To start a [local web server](https://www.polymer-project.org/3.0/docs/tools/polymer-cli-commands#serve) that hosts the demo page and tests:
+To start a [@web/dev-server](https://modern-web.dev/docs/dev-server/overview/) that hosts the demo page and tests:
 
 ```shell
-polymer serve
+npm start
 ```
 
-To lint ([eslint](http://eslint.org/) and [Polymer lint](https://www.polymer-project.org/3.0/docs/tools/polymer-cli-commands#lint)):
+### Linting
 
 ```shell
+# eslint, lit-analyzer, polymer lint and style linting
 npm run lint
+# eslint only
+npm run lint:eslint
+# lit-analyzer only
+npm run lint:lit
+# polymer only
+npm run lint:polymer
 ```
 
-To run unit tests locally using [Polymer test](https://www.polymer-project.org/3.0/docs/tools/polymer-cli-commands#tests):
-
-```shell
-npm run test:polymer:local
-```
-
-To lint AND run local unit tests:
-
+### Testing
 ```shell
+# lint and unit test
 npm test
+# unit tests only
+npm run test:headless
+# debug or run a subset of local unit tests
+# then navigate to `http://localhost:9876/debug.html`
+npm run test:headless:watch
```

Comment:
You sure you grabbed the latest blurb from the create repo? Shouldn't have the localhost:9876 stuff with `web-test-runner` since it opens in a browser for you.

Reply:
Was using a PR to check I'd updated all the needed areas, must have had an old version

Reply:
Sorry yeah the first few PRs I did I hadn't noticed this bit.